### DISCUSSION TITLE
Unable to add network interface

### DISCUSSION THREAD

**Discussion by team4655** [cockpit-project/cockpit#16895] [Q&A] [answered]
Hello,
I am using CentOS 7.9.2009 with Cockpit 195.12.  When I create VM there is no option to add a network interface to the VM.  After the VM is created there still is no "Add Interface" button.
![image](https://user-images.githubusercontent.com/3047658/151733040-4a85bf6a-dab4-4ea0-b6b7-1a719d5ad4f3.png)

Any thoughts?

**Comment by martinpitt** [comment, marked as answer]
This is a really old version which just simply does not have this functionality yet. Please update to CentOS stream 8 or 9 to get a current cockpit-machines version.
